Instructions for running protocol with PyBpod GUI

HenryJFlynn opened the pull request.

HenryJFlynn committed:
Create User Guide

HenryJFlynn committed:
Update and rename User Guide to User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

HenryJFlynn committed:
Update User-Guide.md

ckaraneen commented on User-Guide.md:
```diff
@@ -0,0 +1,153 @@
+# User Guide for Running Mouse2AFC on PyBpod
```
Change this to `Instructions for running Mouse2AFC protocol with PyBpod GUI` or something better.

ckaraneen commented on User-Guide.md:
```diff
@@ -0,0 +1,153 @@
+# User Guide for Running Mouse2AFC on PyBpod
+#### The goal of this document is to provide intructions on how to install and set up pybpod to run the Mouse2AFC protocol.
+#### It will be a copy of the instructions provide by [Pybpod](https://pybpod.readthedocs.io/en/v1.8.1/getting-started/install.html) but with more detail as well as some modification for this use case.
```
Lines [2](https://github.com/ckarageorgkaneen/mouse2afc/pull/6/files#diff-9a566c302cfe9bfb020281f1983851cbf90561ad1ea9f852a88eda3c93f9f87bR2) and [3](https://github.com/ckarageorgkaneen/mouse2afc/pull/6/files#diff-9a566c302cfe9bfb020281f1983851cbf90561ad1ea9f852a88eda3c93f9f87bR2) are redundant. I would delete them.

ckaraneen commented on User-Guide.md:
```diff
@@ -0,0 +1,153 @@
+#### The goal of this document is to provide intructions on how to install and set up pybpod to run the Mouse2AFC protocol.
+#### It will be a copy of the instructions provide by [Pybpod](https://pybpod.readthedocs.io/en/v1.8.1/getting-started/install.html) but with more detail as well as some modification for this use case.
+
+## Prerequisites for Installation
```
Delete this section. We'll just mention the steps in the `Installation` section.

HenryJFlynn committed:
Update User-Guide.md

ckarageorgkaneen commented:
@HenryJFlynn Will try out the instructions today and tomorrow.

ckaraneen committed:
Simplify protocol guide

ckaraneen committed:
Correct guide

HenryJFlynn committed:
Update GUIDE.md

HenryJFlynn committed:
Update GUIDE.md

fix recurring wire1 error (https://github.com/HenryJFlynn/mouse2afc/issues/25#issuecomment-1618184766)

HenryJFlynn committed:
Add pybpod project folder to repo

HenryJFlynn committed:
Update GUIDE.md

add instructions to use pybpod project folder in mouse2afc repo. add general fixes and refinements

ckaraneen committed:
Update pybpod project dir

ckaraneen committed:
Correct GUIDE

HenryJFlynn committed:
Correct pictures

HenryJFlynn committed:
Update GUIDE.md

ckaraneen reviewed (approved):
LGTM
